apps/src/storage/dataBrowser/ColumnHeader.jsx

```diff
@@ -115,33 +116,31 @@ class ColumnHeader extends React.Component {
   }
 
   render() {
-    const containerStyle = [
-      styles.container,
-      {
-        display: this.props.isEditing ? 'none' : null
-      }
-    ];
-    const inputStyle = [
-      dataStyles.input,
-      {
+    const inputStyle = {
+      ...dataStyles.input,
+      ...{
         display: this.props.isEditing ? null : 'none',
         backgroundColor: this.isInputValid() ? null : color.lightest_red,
         minWidth: 80
       }
-    ];
+    };
     return (
       <th style={dataStyles.headerCell} className="uitest-data-table-column">
-        <div style={containerStyle} className="flex">
-          <div style={styles.columnName} className="test-tableNameDiv">
+        <div
+          className={classNames(
+            style.container,
+            this.props.isEditing ? style.containerIsEditing : ''
```

Comment:
Nit: the way I've been seeing this done is `this.props.isEditing && styles.containerIsEditing`

Reply:
Changed!